tests/send-type.js

```diff
@@ -195,7 +194,7 @@ test('polka/send-type::usage (Stream – respect existing header)', t => {
 			t.pass('~> "response" output exists');
 			t.true(info.size > 0, '~~> data piped successfully');
 		} catch (err) {
-			//
+			console.log(err.message)
```

Comment:
this seems like a weird thing to do in a test. if it's an issue then the test should fail instead of just printing to the console